src/interaction/InteractionManager.js

```diff
@@ -761,7 +767,7 @@ export default class InteractionManager extends EventEmitter
      *  interactionEvent, displayObject and hit will be passed to the function
      * @param {boolean} [hitTest] - this indicates if the objects inside should be hit test against the point
      * @param {boolean} [interactive] - Whether the displayObject is interactive
-     * @return {boolean} returns true if the displayObject hit the point
+     * @return {number} returns 1 or 2 if the displayObject hit the point, 0 if not
```

Comment:
How important do you think it is to distinguish between 1 and 2 in the docs here?